www/modules/custom/borg_project_metrics/borg_project_metrics.module

```diff
@@ -84,4 +90,287 @@ function borg_project_metrics_cron() {
+    }
+    elseif ($type == 'commits') {
+      $url = "https://api.github.com/repos/backdrop/backdrop/commits?since=$date_time&page=$i";
+    }
+    elseif ($type = 'comments') {
+      $url = "https://api.github.com/repos/backdrop/backdrop-issues/issues/comments?since=$date_time&page=$i";
+    }
+
+    // Initiate a curl session.
+    $ch = curl_init();
+    curl_setopt($ch, CURLOPT_HTTPHEADER, array('Content-Type: application/json', $authorization));
+    curl_setopt ($ch, CURLOPT_USERAGENT, "Mozilla/4.0 (compatible; MSIE 5.01; Windows NT 5.0)");
+    curl_setopt($ch, CURLOPT_URL, $url);
+    curl_setopt($ch, CURLOPT_RETURNTRANSFER, TRUE);
+    curl_setopt($ch, CURLOPT_CONNECTTIMEOUT, 1);
+    $metrics[$i] = curl_exec($ch);
+    $res_code = curl_getinfo($ch, CURLINFO_RESPONSE_CODE);
+    curl_close($ch);
+    $json = json_decode($metrics[$i]);
+    $i++;
+  } while ($res_code == 200 && !empty($json));
+
+  // If the $type is comments then we also want to get the contrib comments.
+  if ($type == 'comments') {
+    $project_nodes = _get_project_nodes();
+    foreach ($project_nodes as $m) {
+      $project = explode('/', $m[0]);
+      $project = $project[1];
+      if ($project != 'backdrop' && !empty($project)) {
+        $project_comment_metrics = _get_contrib_comments($project, $date_time);
+        $metrics = array_merge($metrics, $project_comment_metrics);
+      }
+    }
+  }
+
+  return $metrics;
+}
+
+/**
+ * Helper function to parse data from githhub api _get_metrics() request and
+ * store the parsed data in backdorp_project_metrics DB table.
+ *
+ * @param string $type
+ * The type {users, commits, comments} of metric to parse.
+ *
+ * @see _get_display_metric_from_db()
+ * @see _get_metrics()
+ * @see _put_metrics_into_db()
+ */
+function _parse_metrics(string $type) {
+  // Get the last recorded data for the metric.
+  $metrics_data = _get_display_metric_from_db($type);
+
+  // Get the current total so we can increment it.
+  $metrics_total = $metrics_data['total_to_date'];
+
+  // Get the last total_to_date date recorded.
+  $last_date = $metrics_data['date'];
+
+  // Query github API for $type since $last_date and count them.
+  $results = _get_metrics($type, $last_date);
+
+  $num = 0;
+  foreach ($results as $r) {
+    $paginated_metrics = json_decode($r);
+    foreach ($paginated_metrics as $metric) {
+      if (!empty($metric)) {
+        $num++;
+      }
+    }
+  }
+  $metrics = $num;
+
+  // Get timestamp.
+  date_default_timezone_set('UTC');
+  $now = date('Y-m-d H:i:s');
+  $total_to_date = $metrics_total + $metrics;
+
+  // Record the metrics in the DB.
+  _put_metrics_into_db($type, $now, $total_to_date);
+}
+
+/**
+ * Implements hook_block_info().
+ */
+function borg_project_metrics_block_info() {
+  $blocks['backdorp_project_metrics'] = array(
+    'info' => t('Backdrop Project Metrics'),
+    'description' => t(
+      'The number of users, commits, and comments to Backdrop
+      since the project started.'
```

Comment:
@serundeputy if you put a line break in a string that needs to be translated, it will cause problems with the translations since the line-break would cause two otherwise identical strings to need to be translated twice. All strings are allowed to exceed the 80 char limit to avoid this problem ;)